WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -41,11 +41,16 @@
               </StackPanel>
             </Button>
             <Button Classes="function"
+                    Classes.pulsate="{Binding !IsSendButtonVisible}"
+                    Classes.active="{Binding IsViewActive}"
```

Comment:
I don't really understand why two classes were required for this. I mean I understand it, but just because I am familiar with the WalletViewModel. Half year later probably I wouldn't know what is the logic here.

Ideally, I can imagine it like `Classes.pulseAnimation={Binding ReceivePulseAnimationEnabled}` or something similar. To be understandable to every developer even at the first glance.

Moreover, if you have 1 variable for it in the ViewModel, it is way easier to manipulate when to enable and when to not.